R/mainTests.r

```diff
@@ -958,6 +959,17 @@ la_col_present <- function(data) {
   return(output)
 }
 
+# Function used to check if each column for that geographic level has any cells that are not blank
```

Comment:
Why is this function (col_completed) now outside of the overcomplete_regional_cols() function?

![image](https://github.com/dfe-analytical-services/dfe-published-data-qa/assets/52536248/66f56af4-b395-4f79-bd3d-b22e1ca473f8)

Reply:
Actually, looks like this was intentional to remove it from the scope of the function so then multiple other functions could use it and save on the copy/pasting of code?

Reply:
Yep, just code neatening. I didn't like it repeating, made the code harder to read.